### ISSUE TITLE
Home/Landing page edits

### ISSUE THREAD

**Issue by billspat** [krishnanlab/geneplexus_app#164]
- In the white frame on the right, increase the padding on the right (from the right edge of the page) to make it equal to the padding on the left (from the left edge of the white frame).

- Add padding to the bottom of the page. Right now, the paper citation is ending at the bottom edge of the page.

- Landing Page Image: Reduce its size (proportionally) so that it occupies just half the width of the white frame. Now, it’s too big.  
Using GenePlexus Section

- Citation style is not consistent on whether it is last name first or first initial first

- Please make the following changes to the text and which parts are hyperlinked:

  - Click ‘Add Genes’ in the panel on the left to start running GenePlexus on your own genes of interest!
  - Watch the [GenePlexus tutorial](https://youtu.be/zEDUqv95vaQ).
  - See the [About page](http://geneplexusdev2202.azurewebsites.net/about) for an overview of how GenePlexus works. 
  - See the [Help page](http://geneplexusdev2202.azurewebsites.net/help) for detailed help on the various inputs and outputs.
  - See an [Example run](http://geneplexusdev2202.azurewebsites.net/jobs/example) where GenePlexus was used to train a model and predict novel genes associated with the disease (see more below). *(currently there’s an extra space at the beginning and the hyperlink runs over to an extra space.)*

- About GenePlexus section, Please change the text to the following:

When a user provides a set of genes to GenePlexus, it trains a custom machine learning (ML) model that captures the patterns of network connectivity of the user’s genes in contrast to other genes in the network. Based on this ML model Geneplexus will:

1. Predict other genes in the network that are similar to the input genes based on their network connectivity.
2. Help interpret the custom trained ML model by comparing it to pre-trained models for various biological processes and diseases, and
3. Provide a visualization of the top predictions in the form of an interactive network graph.
All the results can be downloaded/exported in multiple convenient formats.

**Comment by jacobnewsted** [comment]
these two changes are reflected on the dev branch homepage so I am closing this again
